Makefile

```
@@ -253,6 +257,12 @@ check-release-build: release-build
 	$(info checking for git diff after running 'make release-build')
 	git diff --quiet ; if [ $$? -eq 1 ] ; then echo "Please, commit templates after running 'make release-build' command"; exit 1 ; fi
 
+.PHONY: check-update-helm-sub-charts
+check-update-helm-sub-charts: update-helm-sub-charts
+	$(info checking for git diff after running 'make release-build')
+	git diff --quiet ; if [ $$? -eq 1 ] ; then echo "Please, commit Helm Chart after running 'make update-helm-sub-charts' command"; exit 1 ; fi
```

Consider the following change. This will also won't print `Please, commit Helm Chart after running...` upon success which might be confusing.
```suggestion
	@git diff --quiet || (echo "Error: Please commit Helm Chart changes after running 'make update-helm-sub-charts' command" && exit 1)
```

---

Done

---

For now I just updated the chart. Need to think about a way to check if all sub charts are updated.